tests/fast-integration/logging/loki.js

```diff
@@ -40,9 +41,29 @@ async function checkPersistentVolumeClaimSize() {
   assert.equal(pvc.status.capacity.storage, '30Gi');
 }
 
+function isJsonString(str) {
+  try {
+    JSON.parse(str);
+  } catch (e) {
+    return false;
+  }
+  return true;
+}
+
+async function verifyIstioAccessLogFormat(startTimestamp) {
```

Comment:
Let's also validate that we have typical attributes to fill a "golden signals" dashboard (response codes, sent/received bytes, etc.)